bin/index.mts

```diff
@@ -276,6 +276,16 @@ async function buildPlugin({ watch, noInstall, production, noReload }: Args): Pr
       build.onResolve({ filter: /^replugged(\/\w+)?$/ }, (args) => {
         if (args.kind !== "import-statement") return undefined;
 
+        if (args.path.includes("dist")) {
+          return {
+            errors: [
+              {
+                text: `Unsupported import from dist: ${args.path}`,
```

Comment:
Can you explain the alternative here similar to what we had before?

Reply:
Done